npm/cypress-schematic/src/builders/cypress/index.ts

```diff
@@ -99,8 +99,20 @@ export function startDevServer ({
   devServerTarget,
   watch,
   context }: StartDevServerProps): Observable<string> {
-  const overrides = {
-    watch,
+  // @NOTE: Do not forward watch option if not supported by the target dev server,
+  // this is relevant for running Cypress against dev server target that does not support this option,
+  // for instance @nguniversal/builders:ssr-dev-server.
+  // see https://github.com/nrwl/nx/blob/f930117ed6ab13dccc40725c7e9551be081cc83d/packages/cypress/src/executors/cypress/cypress.impl.ts
+  const angularJson = require(`${context.workspaceRoot }/angular.json`)
+  const { project, target } = targetFromTargetString(devServerTarget)
+  const devServerConfig = angularJson.projects[project].architect[target]
+  let overrides = {}
+
+  if (devServerConfig.builder !== '@nguniversal/builders:ssr-dev-server') {
+    console.info(`Passing watch mode to DevServer - watch mode is ${watch}`)
+    overrides = {
+      watch,
```

Comment:
Instead of making `watch` optional should we just do

```js
coinst overrides = {
  watch: devServerConfig.builder !== '@nguniversal/builders:ssr-dev-server' ? watch : false
}
```

Just a nit/optional update, if we get more things to ignore in the future can do

```ts
const ignoreWatchBuilders = ["@nguniversal/builders:ssr-dev-server"]

overrides = {
  watch: [ignoreWatchBuilders].some(({ builder }) => builder === devServerConfig.builder) ?  false : watch
```

Reply:
watch has to be not there, otherwise the json schema validation will fail - also it doesnt allow additional attributes. 

watch has to be optional